lib/xapi-stdext-unix/unixext.ml

```diff
@@ -501,49 +527,68 @@ let really_write_string fd string =
 
 exception Timeout
 
+let to_milliseconds ms = ms *. 1000. |> int_of_float
+
+(* Allocating a new polly and waiting like this results in at least 3 system calls.
```

Comment:
Needs reformat to break long lines

Reply:
reformat PR here: https://github.com/xapi-project/stdext/pull/82
I then installed `merge-fmt` and rebased this PR as documented here: https://github.com/hhugo/merge-fmt#getting-started
I didn't have to manually solve any merge conflicts and the PR was almost formatted correctly too (when run reformat at the end, there was only one small fixup to the unit test commit, which I squashed into it).

Although nothing has actually changed wrt to this line since it is a comment (and comments aren't reformatted here), so I manually fixed up the comment